### DISCUSSION TITLE
Factory used in OpenAI's ChatGPT application!

### DISCUSSION THREAD

**Discussion by hmlongco** [hmlongco/Factory#110] [Announcements]
A friend at Google just pointed out that Factory is apparently being used in OpenAI's ChatGPT application for iOS.

![image](https://github.com/hmlongco/Factory/assets/709283/83c18baf-7b26-4164-8e3c-bfecf76a5554)

Do you use Factory in your application? Send me a link or screenshot and I'll mention it here!

**Comment by alkincakiralar1996** [comment]
As Fizbot, we also use your library in our project, and I also recommend my team follow your articles regularly.

Here is the link below

https://apps.apple.com/tr/app/fizbot/id1267959556?l=tr&platform=iphone
